ext/curl/config.m4

```diff
@@ -6,75 +6,97 @@ PHP_ARG_WITH(curl, for cURL support,
 [  --with-curl[=DIR]         Include cURL support])
 
 if test "$PHP_CURL" != "no"; then
-  if test -r $PHP_CURL/include/curl/easy.h; then
-    CURL_DIR=$PHP_CURL
```

Comment:
> should we use pkg-config even if the path is specified explicitly?

Yes, this is what is done, using libcurl.pc in specified directory

     elif test -r $PHP_CURL/lib/pkgconfig/libcurl.pc; then